### ISSUE TITLE
Maximize column

### ISSUE THREAD

**Issue by TeddyDD** [crazko/krystal#6]
Feature request:

Option for maximizing column (so it takes full screen width) would be useful for writing longer tiddlers (especially with preview since TW show preview in vertical split)

**Comment by crazko** [comment]
I was also thinking about this feature, though in a different context (to show linked graph of tiddlers).

It could be hard to implement it with active scrollbar, but having it static, without scrollbar while maximized should be OK.

What do you think?

**Comment by TeddyDD** [comment]
Yeah, I agree that while maximized you probably don't need scrollbar - especially if keyboard shortcut could be used to toogle maximized state.

**Comment by keelancook** [comment]
Let me start by saying, I've been using your theme since you dropped it, and I still love it. This theme with the added capability of the [Streams plugin](https://saqimtiaz.github.io/sq-tw/streams.html) is a winning combination.

Though, I'd also like to see (1) the ability to maximize a pane and (2) adjust the width of all panes.

It's easy enough to get into the tiddlers in the plugin to adjust the width, but you might consider making it configurable in the theme tweaks section. As for maximizing, since you've made it responsive enough, if you narrow the browser window to the right spot, it essentially maximizes that pane. This isn't the most polished approach, but it's what I'm currently doing if I want to focus on one tiddler for longer writing. 

One last thing, I noticed you mentioned showing a linked graph of tiddlers. Are you planning on attempting an implementation of that in this theme? If so, that would be dynamite. I've wanted to try and shoehorn tiddlymap into it, but I've not had the time (nor do I know enough about fiddling with this stuff). But, I would love to see a Roam-esque graph view as a tiddler that you could call up through search and maximize to take the full pane. 

You've done a great service for the TW5 community with this, by the way.

**Comment by crazko** [comment]
Thanks @keelancook, I'm glad you like it.

> It's easy enough to get into the tiddlers in the plugin to adjust the width, but you might consider making it configurable in the theme tweaks section.

It's feasible. Can you, please, make a new issue, so we can track it separately? When I finish other pending things, I'll try to look at it.

> One last thing, I noticed you mentioned showing a linked graph of tiddlers. Are you planning on attempting an implementation of that in this theme?

I'm not sure, whether it should be a part of this plugin, or a new one. It may depend on the necessary dependencies, as I don't want to have it very bloated.

To give you an example of what I am playing with, try to create a new tiddler with this content:

```
<pre>digraph G {
<$list filter="[all[]!is[system]]" variable="a"><$list filter="[<a>backlinks[]]" variable="b">  "<<b>>" -> "<<a>>";
</$list></$list>}</pre>
```

And then copy-paste the result here https://visjs.github.io/vis-network/examples/network/data/dotLanguage/dotPlayground.html :)

**Comment by keelancook** [comment]
Happy to make a new issue, I'll do so soon.

As for the graph, I'm sold. There's some real potential with what you're working on currently. I agree that it might be better served as its own plugin. One could use your Krystal theme with or without it. Additionally, they could implement your graph in a different theme that way. I think many people would be interested in a graph view that is easier to implement (and cleaner stylistically) than what TiddlyMap currently provides.

I believe the ideal instance would be called up as a tiddler itself in the story river (especially if the tiddler can be maximized in some way). Since you don't have a true sidebar in the Krystal theme, this would make the most sense with your setup. However, I would even prefer it open in the story river in other themes instead what is typical with graphs like TiddlyMap, a tab in a very large and odd-looking sidebar.

Excited to see how this develops! Again, thanks for all of your work. I'm currently neck deep in an academic research writing project and Krystal has been a refreshing way to tackle the work.

**Comment by keelancook** [comment]
Agreed. Clean, simple implementation for maximizing. Helps a ton. 

Still my favorite theme!

**Comment by crazko** [comment]
Thanks!
